File: calico-us-berkeley/2024/problems/toki/templates/toki_template.cpp
#include <iostream>

using namespace std;

/**
 * Determine if the word W follows the rules of toki pona phonotactics.
 * 
 * W: the word to check
 */
string solve(string W) {
    // YOUR CODE HERE
    return "pona";
}

int main() {
    int T;
    cin >> T;
    for (int i = 0; i < T; i++) {
        string W;
        cin >> W;
        cout << solve(W) << '\n';
    }
}

File: calico-us-berkeley/2024/problems/toki/templates/toki_template.java
import java.io.*;

class Solution {
    /**
     * Determine if the word W follows the rules of toki pona phonotactics.
     * 
     * W: the word to check
     */
    static String solve(String W) {
        // YOUR CODE HERE
        return "pona";
    }
    
    static BufferedReader in = new BufferedReader(new InputStreamReader(System.in));
    static PrintWriter out = new PrintWriter(System.out);

    public static void main(String[] args) throws IOException {
        int T = Integer.parseInt(in.readLine());
        for (int i = 0; i < T; i++) {
            String W = in.readLine();
            out.println(solve(W));
        }
        out.flush();
    }
}
